treeherder/log_parser/failureline.py

```diff
@@ -138,7 +138,14 @@ def create_failure_line(job_log, failure_line):
 
 
 def create(job_log, log_list):
-    failure_lines = [create_failure_line(job_log, failure_line) for failure_line in log_list]
+    for failure_line in log_list:
+        if failure_line.action not in FailureLine.ACTION_LIST:
+            newrelic.agent.record_custom_event("unsupported_failure_line_action", failure_line)
```

Comment:
I wonder if we'd want to raise a `ValidationError` and then catch that except and log it. I was looking in the docs and it doesn't seem like a that type of error is raised by Django if a choice isn't valid: https://docs.djangoproject.com/en/3.0/ref/models/fields/#choices. We might have to manually do something like that here or on the model.

But this also looks good :)

Reply:
What if I just log a `ValueError` here?  I could `raise` and `catch` it right here, but not sure of the value over just logging it.  How does this change look?

Reply:
Logging it for now seems fine, but this does seem like something we'd want to do on the model level so the validation/raising of a ValueError occurs regardless of where the table is queried.

Reply:
Yeah, that's a good point.  Created Bug 1641304 for this.